src/redis.c

```diff
@@ -353,6 +353,41 @@ static int redis_handle_query(redisContext *rh, redis_node_t *rn,
   return 0;
 } /* }}} int redis_handle_query */
 
+static int redis_db_stats(char *node, char const *info_line) /* {{{ */
+{
+  unsigned char db;
+  unsigned char max_db = 16;
+  static char field_name[10];
+  static char db_id[3];
+  char *str;
+  static char buf[MAX_REDIS_VAL_SIZE];
+  value_t val;
+  int i;
+
+  for (db = 0; db < max_db; db++) {
```

Comment:
Nit, at your option: the currently recommended coding style is to declare the loop variable as part of the `for` loop, i.e.:

```c
for (int db = 0; db < max_db, db++) {
```